semgrep/semgrep-docs
# add morehelp to docitem/content

# Thanks for improving Semgrep Docs 😀

### Please ensure

- [x] A technical writer reviews the content or PR

### Previously, on semgrep-docs...

- We had to import `<MoreHelp />` for every doc that we made.
- We had to put the `<ul>` to render tags

### Now, on semgrep-docs...

- `<MoreHelp />` is part of the document template and is baked into every document automatically.
- So that it doesn't get repeated, I have removed all the imports of it.
- Same with the opening tag `<ul>`

### How to test

- Pull this PR and try out some of the pages. It's programmatic find-and-replace so if some work they should all work.
